metaflow/metaflow_environment.py

```diff
@@ -126,6 +126,7 @@ def get_package_commands(
             "after 6 tries. Exiting...' && exit 1; "
             "fi" % code_package_url,
             "TAR_OPTIONS='--warning=no-timestamp' tar xf job.tar",
+            f"mflog {code_package_url=}",
```

Comment:
Nit: maybe printing this on line 120 as `Downloading code package from {code_package_url}` looks better to users?